### ISSUE TITLE
Assign EIP to CCE cluster not possible

### ISSUE THREAD

**Issue by 0xd3e** [opentelekomcloud/terraform-provider-opentelekomcloud#311]
Hi,

I want my Kubernetes cluster (CCE) to be available from the Internet. Therefor I need to assign an EIP to the cluster, which is possible via the OTC dashboard, but not via this Terraform provider.

### Terraform Version

```
> terraform -v
Terraform v0.12.2
+ provider.opentelekomcloud v1.9.0
```

### Affected Resource(s)

- opentelekomcloud_vpc_eip_v1
- opentelekomcloud_cce_cluster_v3

### Expected Behavior

I'd expect that the resource `opentelekomcloud_cce_cluster_v3` has an attribute like `port_id` that can be used to assign an EIP to the cluster. This is possible via the OTC dashboard.

### Actual Behavior

No useful attribute is provided by `opentelekomcloud_cce_cluster_v3` in this scenario.

**Comment by niuzhenguo** [comment]
@danieltrautmann I'm sorry that we can't fetch `port_id` information through API, so it's not possible to achieve your request on terraform side. Maybe you can contact a support on Cloud side to make them expose such info.

**Comment by niuzhenguo** [comment]
fixed by #400
